tests/test_client_state.py

```diff
@@ -1,23 +1,51 @@
-from nicegui import ui, app
+import pytest
+
+from nicegui import ui, app, context
 from nicegui.testing import Screen
 
 
 def test_session_state(screen: Screen):
-    app.storage.client['counter'] = 123
-
     def increment():
         app.storage.client['counter'] = app.storage.client['counter'] + 1
 
-    ui.button('Increment').on_click(increment)
-    ui.label().bind_text(app.storage.client, 'counter')
+    @ui.page('/')
+    async def page():
+        await context.get_client().connected()
```

Comment:
Why is waiting for client connection needed here?

Reply:
Fixed. You are right, in difference to .tab - even though the  data is of course bound to the Client it of course does not need to be established yet.